cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -877,6 +879,24 @@ func (k *KubernetesClient) isResourceNamespaced(gvk schema.GroupVersionKind) (bo
 	return false, errors.New(fmt.Sprintf("group version kind not found: %v", gvk.String()))
 }
 
+func (k *KubernetesClient) role(group, version, kind, name, namespace string ) (*dto.Role, error){
+	 roleList, err := k.clientset.RbacV1().Roles(namespace).Get(context.Background(), name, metav1.GetOptions{})
```

Comment:
@tanbirali can you remove the variable not the func name

Reply:
Do you mean the roleList variable?